docs/content/quickstart.md

```diff
@@ -3,116 +3,127 @@ title: QuickStart Guide
+* Registry - An OCI compliant artifact store that allows machine image management. 
 
-Here is a very basic **porter.yaml** file:
+## Inspect a bundle
 
-```yaml
-name: HELLO
-version: 0.1.0
-description: "An example Porter configuration"
-tag: getporter/porter-hello
+Before using a bundle that you've found, you can inspect a bundle to see more information about it with `porter inspect`. By inspecting the bundle you can see all images that will be used as a result of bundle actions. 
 
-mixins:
-  - exec
+```
+porter inspect --tag getporter/porter-hello:v0.1.0
+```
 
-install:
-  - exec:
-      description: "Install Hello World"
-      command: bash
-      flags:
-        c: echo Hello World
+Sample output:
+```
+Name: HELLO
+Description: An example Porter configuration
+Version: 0.1.0
 
-upgrade:
-  - exec:
-      description: "World 2.0"
-      command: bash
-      flags:
-        c: echo World 2.0
+Invocation Images:
+Image                                                                                            Type     Digest                                                                    Original Image
+getporter/porter-hello@sha256:678f5726e2d79263a4ac3f02a35eaf41a312aa833c6f55afa45155730db375a7   docker   sha256:678f5726e2d79263a4ac3f02a35eaf41a312aa833c6f55afa45155730db375a7   getporter/porter-hello-installer:0.1.0
 
-uninstall:
-  - exec:
-      description: "Uninstall Hello World"
-      command: bash
-      flags:
-        c: echo Goodbye World
+Images:
+No images defined
 ```
 
-## Build the bundle
+With this information, you can pull, inspect, and vet images before you use the bundle. In this example, you are inspecting the HELLO bundle with the tag  getporter/porter-hello:v0.1.0. There are no referenced images. 
 
-The `porter build` command will generate the bundle:
+## Run the install instructions from a bundle located in a registry
+
+To install a bundle, you use the `porter install` command. 
 
 ```
-porter build
+porter install --tag getporter/porter-hello:v0.1.0
+
 ```
 
-## Install the bundle
+In this example, you are installing athe 0.1.0 version of the bundle from the default registry by specifying a tag. 
+
+## List 
 
-You can then use `porter install` to install your bundle:
+To see the bundle installations, use the `porter list` command. 
 
 ```
-porter install
+porter list
 ```
 
-If you wish to uninstall the bundle, you can use `porter uninstall`:
-
+Sample Output:
 ```
-porter uninstall
+NAME       CREATED          MODIFIED         LAST ACTION   LAST STATUS
+HELLO      21 minutes ago   21 minutes ago   install       succeeded
 ```
 
-## Publish the bundle
+In this example, it shows the bundle metadata along with the creation time, modification time, the last action that was performed, and the status of the last action. The default name of the bundle, HELLO, is used since there was no installation name specified at the command line. 
```

Comment:
There isn't any bundle metadata displayed in the command above, how about this?

```suggestion
In this example, it shows the installation name along with the creation time, modification time, the last action that was performed, and the status of the last action. The default name of the bundle, HELLO, is used since there was no installation name specified at the command line. 
```